### PR TITLE
Add ISO/IEC Directives, editions 10-13

### PR DESCRIPTION
**Pull request by anermina**
As requested in #5 . 

Note: `:doctype:` was initially set to `international-standard`, but these documents were converted with `directive` in https://github.com/metanorma/mn-samples-iso/tree/main/sources/directives, so I applied this `:doctype:`.

### CODE REVIEW

File: data/iso_iec_dir_1_iso_sup_ed-11-2020.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- type: title-part
+  content: Procédures spécifiques à l'ISO
+  language: fr
+  script: Latn
+
+part: "1"
+
+link:
+- type: src
+  content: https://fdocuments.in/document/isoiec-directives-part-1-consolidated-iso-supplement-and-iso-proceduresiso.html
```
Comment (ronaldtse) [inline data/iso_iec_dir_1_iso_sup_ed-11-2020.yml]: @anermina let's remove all non-authoritative sources (the only authoritative source is iso.org). Rather have them empty.

File: data/iso_iec_dir_1_iso_sup_ed-11-2020.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- type: obp
+  content: https://fdocuments.in/embed/v1/isoiec-directives-part-1-consolidated-iso-supplement-and-iso-proceduresiso.html
```
Comment (ronaldtse) [inline data/iso_iec_dir_1_iso_sup_ed-11-2020.yml]: The OBP is the ISO Online Browsing Platform. The links start with https://www.iso.org/obp. There is no need to fill this in if it is not from the OBP.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
Also, the tests are failing for these files:

```
##[group]Run bundle exec ruby check_data.rb

Parsing data/iso_iec_dir_1_iso_sup_ed-10-2019.yml failed. Parsed content doesn't match to source.
part:
  - 1
  + 

Parsing data/iso_iec_dir_1_iso_sup_ed-10-2019.yml failed. No primary id.
Parsing data/iso_iec_dir_1_iso_sup_ed-11-2020.yml failed. Parsed content doesn't match to source.
part:
  - 1
  + 

Parsing data/iso_iec_dir_1_iso_sup_ed-11-2020.yml failed. No primary id.
Parsing data/iso_iec_dir_1_iso_sup_ed-12-2021.yml failed. Parsed content doesn't match to source.
part:
  - 1
  + 

Parsing data/iso_iec_dir_1_iso_sup_ed-12-2021.yml failed. No primary id.
Parsing data/iso_iec_dir_1_iso_sup_ed-13-2022.yml failed. Parsed content doesn't match to source.
part:
  - 1
  + 

Parsing data/iso_iec_dir_1_iso_sup_ed-13-2022.yml failed. No primary id.
##[error]Process completed with exit code 1.
```

**Comment by anermina** [comment]
@ronaldtse thank you for explanation! 
Changes are applied.

I've noticed we don't have part number encoded anywhere in these YAML files. First checks didn't pass because I added line `part: "1"` so I removed it. Is there a way to encode a part number?

**Comment by ronaldtse** [comment]
@andrew2net can you help advise @anermina how to encode a part number here? But then there is the "ISO Supplement" to "Part 1". What do we do about this?

**Comment by andrew2net** [comment]
@anermina we don't have a `part` attribute in the Relaton model. In this case, we can use `edition` as we need to handle multiple editions.
```yaml
edition: "1"
```

**Comment by ronaldtse** [comment]
@andrew2net the "part" number is not "edition". An edition is the same "work" that the newer edition supersedes the older edition. The fact that a "part" is not in Relaton may be an issue. Ping @opoudjis .

**Comment by ronaldtse** [comment]
I've discussed with @opoudjis that currently there is no easy way to correctly encode a part number right now.

There are two things we can do today:
1. Encode a [partOf document relationship](https://github.com/relaton/relaton-models/blob/ec12a0b5f00fd9e5dd19a63e6d0e1658d0165331/grammars/biblio.rnc#L403) for a Part and point to the "ISO/IEC Directives" (to identify it is part of the "ISO/IEC Directives"). In the future we can make possible a way to add a "part identifier" to this relationship.
2. Include the Part number as a string the title. The drawback is we cannot extract that out using data relationships.

**Comment by andrew2net** [comment]
@anermina @ronaldtse usually ISO references include part numbers. For example `ISO 19115-1:2014` contains document number `19115` and part number `1`. The ref `ISO 19115-2:2019` has the part number `2`.

> 1. Encode a [partOf document relationship](https://github.com/relaton/relaton-models/blob/ec12a0b5f00fd9e5dd19a63e6d0e1658d0165331/grammars/biblio.rnc#L403) for a Part and point to the "ISO/IEC Directives" (to identify it is part of the "ISO/IEC Directives"). In the future we can make possible a way to add a "part identifier" to this relationship.

The implementation of the `partOf` relation in YAML format is like:
```yaml
...
relation:
  - type: partOf
    bibitem:
      formattedref: ISO 19115:2003
      docid:
        - id: ISO 19115:2003
          type: ISO
          primary: true
....
```

**Comment by ronaldtse** [comment]
@andrew2net I've entered the additional content. The specs are failing now. Can you have a look? Thanks!

**Comment by ronaldtse** [comment]
Thanks @andrew2net @anermina !

**Pull request merged**